cookbooks/particle_injection.prm

```diff
@@ -0,0 +1,247 @@
+  set Tangential velocity boundary indicators = bottom, top, left, right
+
+  # The final part of this section describes whether we
+  # want to include adiabatic heating (from a small
+  # compressibility of the medium) or from shear friction,
+  # as well as the rate of internal heating. We do not
+  # want to use any of these options here:
+
+  set Include adiabatic heating               = false
+  set Include shear heating                   = false
+end
+
+
+# The following two sections describe first the
+# direction (vertical) and magnitude of gravity and the
+# material model (i.e., density, viscosity, etc). We have
+# discussed the settings used here in the introduction to
+# this cookbook in the manual already.
+
+# HL: A Rayleigh number of 1E5
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 10   # = Ra / Thermal expansion coefficient
+  end
+end
+
+
+subsection Material model
+  set Model name = simple # default:
+
+  subsection Simple model
+    set Reference density             = 1
+    set Reference specific heat       = 1
+    set Reference temperature         = 0
+    set Thermal conductivity          = 1
+    set Thermal expansion coefficient = 1
+    set Viscosity                     = 1E-3
+  end
+end
+
+
+# We here use a globally refined mesh without
+# adaptive mesh refinement.
+
+subsection Mesh refinement
+  set Initial global refinement          = 0
+  set Initial adaptive refinement        = 0
+  set Time steps between mesh refinement = 0
+end
+
+
+# The final part is to specify what ASPECT should do with the
+# solution once computed at the end of every time step. The
+# process of evaluating the solution is called `postprocessing'
+# and we choose to compute velocity and temperature statistics,
+# statistics about the heat flux through the boundaries of the
+# domain, and to generate graphical output files for later
+# visualization. 
+
+subsection Postprocess
+   set List of postprocessors = basic statistics, visualization, tracers, heat flux statistics, velocity statistics
+#  set List of postprocessors = basic statistics, visualization, heat flux statistics, velocity statistics
+#  set List of postprocessors = basic statistics, heat flux statistics, velocity statistics
+#  set List of postprocessors = basic statistics, heat flux statistics, velocity statistics, tracers
+
+   subsection Visualization
+     set Output format                 = vtu
+     set Number of grouped files       = 1
+     set Time between graphical output = 0
+   end
+
+  subsection Tracers
+    set Number of tracers = 1440 # = 45*32
+    set Time between data output = 0
+    set Data output format = vtu
+
+    set Particle generator name = uniform box
+    set Integration scheme = rk2
+
+    set List of tracer properties = initial position, position, velocity
+
+    subsection Generator
+      # Prevent particles from being placed on the boundary layer within a 
```

Comment:
Which generator is actually used for this test? Supposedly the default one? Could you still add that line here, in case the default gets changed at some point?